plugins/queue/scheduler.js

```diff
@@ -688,6 +689,32 @@ async function clearCache(executor, config) {
     }
 }
 
+/**
+ * Pushes a message to unzip artifacts
+ * @async unzipArtifacts
+ * @param {Object} executor
+ * @param {Object} config               Configuration
+ * @param {String} config.buildId       Unique ID for a build
+ */
```

Comment:
It seems that you forgot to write the JSDoc of the return value.

Reply:
Thx.  I fixed it.